### ISSUE TITLE
Horizontal mode?

### ISSUE THREAD

**Issue by keichan34** [kolinkrewinkel/KKGridView#39]
Is it possible to add a horizontal mode as well (left-to-right / right-to-left)? I can imagine some cases in which this would be really useful if it had a vertical and horizontal mode..

**Comment by kemenaran** [comment]
I am also interested in an horizontal mode.

I may start hacking the code to add support for this. Do you have any ideas on how implement this API-wise ? I think its important to keep the API clean and simple — even if supporting horizontal mode will add some complexity.

**Comment by SpacyRicochet** [comment]
Our current GridView implementation uses a `GridLayoutOrientation` property which can be set by using enum values `GridLayoutVertical` and `GridLayoutHorizontal`.
Making sure the default is `GridLayoutVertical` ensures that everyone that isn't interested in a horizontal layout won't be confused and everyone that is interested can make it happen by calling a single setter.

Would this be a simple enough solution?

**Comment by keichan34** [comment]
I think that sounds pretty good... Is there an option for left-to-right or right-to-left as well?

**Comment by SpacyRicochet** [comment]
Just to clarify, I'm not talking about an implementation of KKGridView, just our own code which accomplishes the same. I was just suggesting a way for the API of KKGridView to be kept simple.

**Comment by kolinkrewinkel** [comment]
Yes, I definitely wanted to add this.  Will definitely be pursuing it personally once I'm done with the main implementation.

**Comment by rantav** [comment]
I'm also interested in vertical scrolling. If not implemented here, could anyone recommend another grid view that does support that? (AQGridView supports that but it's buggy)

**Comment by SpacyRicochet** [comment]
I think we've got vertical scrolling down already ;)

**Comment by rantav** [comment]
oh yeah, of course I meant that I need Horizontal scrolling of course...
